code/game/objects/effects/decals/decal.dm

```diff
@@ -15,7 +15,7 @@
 		qdel(src)
 
 /obj/effect/decal/proc/NeverShouldHaveComeHere(turf/T)
-	return isspaceturf(T) || isclosedturf(T) || islava(T) || istype(T, /turf/open/water) || ischasm(T)
+	return isclosedturf(T) || isgroundlessturf(T)

```

Comment:
)

Reply:
wait what where do i put the close parenthesis

Reply:
nowhere.